src/ray/core_worker/core_worker.cc

```diff
@@ -354,6 +400,19 @@ void CoreWorker::RunIOService() {
   io_service_.run();
 }
 
+void CoreWorker::OnNodeRemoved(const rpc::GcsNodeInfo &node_info) {
+  const auto node_id = ClientID::FromBinary(node_info.node_id());
+  RAY_LOG(INFO) << "Node failure " << node_id;
+  const auto lost_objects =
+      memory_store_->GetAndDeletePlasmaObjectsOnRemovedNode(node_id);
+  reference_counter_->ResetDeleteCallbacks(lost_objects);
+  for (const auto &object_id : lost_objects) {
+    RAY_LOG(INFO) << "Object " << object_id << " lost due to node failure " << node_id;
```

Comment:
There could be quite a lot of objects on a given node, how about using `DEBUG` here?

Reply:
Actually, I think this is probably okay because it's `INFO`, not `WARNING`, so it won't go to the driver. It's important to have this information in the backend logs by default.